source-dynamodb/.snapshots/TestDiscovery-additional_table_without_stream_enabled

```diff
@@ -0,0 +1,96 @@
+Binding 0:
+{
+    "recommended_name": "discoverTable",
+    "resource_config_json": {
+      "table": "discoverTable"
+    },
+    "document_schema_json": {
+      "$defs": {
+        "discoverTable": {
+          "type": "object",
+          "required": [
+            "partitionKey"
```

Comment:
should we use `/_meta/partitionKey` or even `/_meta/key` ?

The partition key isn't _literally_ part of the document, as I understand, so it would be a bit more consistent in the sense that "connector-added properties go under /_meta". It also roughly matches the behavior of other schema-less connectors that key off of /_meta/...

It believe this will work w/ end-to-end materializations, despite `/_meta` filtering from recommended fields, because it's the collection key.

Reply:
The partition key is part of the document - it's like any of the other columns in the DynamoDB record, except it has a schema applied to it, so it is always required to be the same type and will always be there with the record. I called it `partitionKey` for this test ([ref](https://github.com/estuary/connectors/pull/877/files#diff-1e8f7e60698434a0b474287b9be50f4d4862cd6bc4591fb54586fcfd739b12fcR27)), but it could be called anything, and would almost always be a relevant property to include in the document.

Reply:
:+1: i retract my comment then